From 37de362cd2852057e08123920c6f14fac2f02ba4 Mon Sep 17 00:00:00 2001
From: David Runge <dave@sleepmap.de>
Date: Mon, 9 Jan 2023 22:13:32 +0100
Subject: [PATCH] Update build-system dependencies

pyproject.toml:
Update setuptools-scm to >= 7.1.0 to drop the requirement for
setuptools-scm-git-archive (since the functionality is included since
7.0.0).
Update requirement for setuptools accordingly.
---
 pyproject.toml | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index f53bdb2..768de59 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,11 +1,10 @@
 [build-system]
 requires = [
   # Essentials
-  "setuptools>=40.6.0",
+  "setuptools>=45",
 
   # Plugins
-  "setuptools_scm[toml] >= 3.5",  # version is required for "no-local-version" scheme + toml extra is needed for supporting config in this file
-  "setuptools_scm_git_archive >= 1.1",
+  "setuptools_scm[toml] >= 7.1.0",  # version is required for "no-local-version" scheme + toml extra is needed for supporting config in this file
 ]
 build-backend = "setuptools.build_meta"
 
-- 
2.39.0

